### ISSUE TITLE
ToM and CAT computer set up documentation

### ISSUE THREAD

**Issue by eduongAZ** [ml4ai/tomcat#265]

**Comment by adarshp** [comment]
@eduongAZ  - #288 has been resolved, please add the documentation to the docs/ folder in the repo, preferably as a .rst (ReStructuredText) file. Thanks!

**Comment by eduongAZ** [comment]
During the Monday meeting, it seems that we need to set up an internal documentation system to not share sensitive information to the public.

**Comment by kobus-barnard** [comment]
Yes. Finding a place for internal documentation  is on my plate.
